crates/loro-internal/src/handler.rs

```diff
@@ -1438,6 +1469,93 @@ impl TextHandler {
         Ok(override_styles)
     }
 
+    /// If attr is specified, it will be used as the attribute of the inserted text.
+    /// It will override the existing attribute of the text.
+    fn insert_with_txn_and_attr_utf8(
```

Comment:
We can reuse `insert_with_txn_and_attr` inside the implementation of this function